rust/patchable/src/main.rs

```diff
@@ -263,8 +269,19 @@ fn main() -> Result<()> {
     .context(ConfigureGitLoggingSnafu)?;
 
     let opts = <Opts as clap::Parser>::parse();
-    let images_repo = repo::discover_images_repo(".").context(FindImagesRepoSnafu)?;
-    let images_repo_root = images_repo.workdir().context(NoImagesRepoWorkdirSnafu)?;
+    let images_repo_root_pathbuf = match opts.images_repo_root {
+        Some(path) => {
+            if !path.is_dir() {
+                return ImagesRepoRootDirCheckSnafu { path }.fail();
+            }
```

Comment:
Not sure we actually need this validation - any actual file access will fail anyway (and I'd say that, for something like `init`, this is a perfectly fine place to be in).